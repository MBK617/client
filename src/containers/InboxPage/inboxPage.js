import React from 'react';
import { View, Text } from "react-native";

const InboxPage = () => {
  return (
    <View>
      <Text>Inbox</Text>
    </View>
  );
}

export default InboxPage;