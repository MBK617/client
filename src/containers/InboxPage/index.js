import InboxPage from './inboxPage';
export default InboxPage;