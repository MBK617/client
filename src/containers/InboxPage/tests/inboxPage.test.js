import React from 'react';
import { render, cleanup } from '@testing-library/react-native';

import InboxPage from '..';

describe('<InboxPage />', () => {
  let rendered;
  beforeEach(() => {
    rendered = render(<InboxPage />);
  })

  afterEach(cleanup);

  it('should render correctly', () => {
    const tree = rendered.toJSON();
    expect(tree).toMatchSnapshot();
  });
});