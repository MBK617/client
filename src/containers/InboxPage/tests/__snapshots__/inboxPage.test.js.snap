// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InboxPage /> should render correctly 1`] = `
<View>
  <Text>
    Inbox
  </Text>
</View>
`;
